packages/dd-trace/src/plugins/index.js

```diff
@@ -1,75 +1,15 @@
-  get 'ioredis' () { return require('../../../datadog-plugin-ioredis/src') },
-  get 'jest-circus' () { return require('../../../datadog-plugin-jest/src') },
-  get 'jest-config' () { return require('../../../datadog-plugin-jest/src') },
-  get 'jest-environment-node' () { return require('../../../datadog-plugin-jest/src') },
-  get 'jest-environment-jsdom' () { return require('../../../datadog-plugin-jest/src') },
-  get 'jest-jasmine2' () { return require('../../../datadog-plugin-jest/src') },
-  get 'jest-worker' () { return require('../../../datadog-plugin-jest/src') },
-  get 'koa' () { return require('../../../datadog-plugin-koa/src') },
-  get 'koa-router' () { return require('../../../datadog-plugin-koa/src') },
-  get 'kafkajs' () { return require('../../../datadog-plugin-kafkajs/src') },
-  get 'mariadb' () { return require('../../../datadog-plugin-mariadb/src') },
-  get 'memcached' () { return require('../../../datadog-plugin-memcached/src') },
-  get 'microgateway-core' () { return require('../../../datadog-plugin-microgateway-core/src') },
-  get 'mocha' () { return require('../../../datadog-plugin-mocha/src') },
-  get 'mocha-each' () { return require('../../../datadog-plugin-mocha/src') },
-  get 'moleculer' () { return require('../../../datadog-plugin-moleculer/src') },
-  get 'mongodb' () { return require('../../../datadog-plugin-mongodb-core/src') },
-  get 'mongodb-core' () { return require('../../../datadog-plugin-mongodb-core/src') },
-  get 'mysql' () { return require('../../../datadog-plugin-mysql/src') },
-  get 'mysql2' () { return require('../../../datadog-plugin-mysql2/src') },
-  get 'net' () { return require('../../../datadog-plugin-net/src') },
-  get 'next' () { return require('../../../datadog-plugin-next/src') },
-  get 'oracledb' () { return require('../../../datadog-plugin-oracledb/src') },
-  get 'openai' () { return require('../../../datadog-plugin-openai/src') },
-  get 'paperplane' () { return require('../../../datadog-plugin-paperplane/src') },
-  get 'pg' () { return require('../../../datadog-plugin-pg/src') },
-  get 'pino' () { return require('../../../datadog-plugin-pino/src') },
-  get 'pino-pretty' () { return require('../../../datadog-plugin-pino/src') },
-  get 'playwright' () { return require('../../../datadog-plugin-playwright/src') },
-  get 'redis' () { return require('../../../datadog-plugin-redis/src') },
-  get 'restify' () { return require('../../../datadog-plugin-restify/src') },
-  get 'rhea' () { return require('../../../datadog-plugin-rhea/src') },
-  get 'router' () { return require('../../../datadog-plugin-router/src') },
-  get 'sharedb' () { return require('../../../datadog-plugin-sharedb/src') },
-  get 'tedious' () { return require('../../../datadog-plugin-tedious/src') },
-  get 'winston' () { return require('../../../datadog-plugin-winston/src') }
+const plugins = require('./pluginList')
+
+// Adds plugins for the modules without a node:* prefix to the plugin list
```

Comment:
Is this magic really needed? It just seems to make things harder to read.

Reply:
Can you elaborate?

Reply:
Well I think it's because this approach looks a little cleaner since adding node:* specifiers along with their non node specifier names makes the list longer but we're just adding new names yet the instrumentations are still the same